mono/mini/alias-analysis.c

```diff
@@ -211,7 +213,9 @@ lower_memory_access (MonoCompile *cfg)
 			case OP_STOREI2_MEMBASE_REG:
 			case OP_STOREI4_MEMBASE_REG:
 			case OP_STOREI8_MEMBASE_REG:
+#ifndef MONO_ARCH_SOFT_FLOAT_FALLBACK
 			case OP_STORER4_MEMBASE_REG:
+#endif
```

Comment:
for R8 is ok because both in the jit and in memory they have the same size (for R4 we had 2 int vregs, so 8 bytes in jit, and 4 bytes in memory)